src/Fizzler.csproj

```diff
@@ -26,23 +16,21 @@
     <PackageTags>selectors w3c</PackageTags>
     <PackageIconUrl></PackageIconUrl>
     <PackageOutputPath>..\dist</PackageOutputPath>
+    <IncludeSymbols>true</IncludeSymbols>
+    <IncludeSource>true</IncludeSource>
     <Copyright>Copyright © 2009 Atif Aziz, Colin Ramsay. All rights reserved. Portions Copyright © 2008 Novell, Inc.</Copyright>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
     <EmbedUntrackedSource>true</EmbedUntrackedSource>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
     <OutputPath>..\bin\Debug\</OutputPath>
-    <!-- to embed complete (full) pdbs in Debug builds. -->
-    <DebugType>embedded</DebugType>
-    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
     <OutputPath>..\bin\Release\</OutputPath>
-    <!-- to embed complete pdbs in Release builds as well. -->
```

Comment:
Uh why was this reverted?

Reply:
I intend to restore this, but via **[DotNet.ReproducibleBuilds]**, which [defaults to `DebugType` of `embedded` if not specified](https://github.com/dotnet/reproducible-builds/blob/7ae5c7db58eff2a57b556e3f56b2d5e5143518d5/src/DotNet.ReproducibleBuilds/DotNet.ReproducibleBuilds.props#L5).

[DotNet.ReproducibleBuilds]: https://www.nuget.org/packages/DotNet.ReproducibleBuilds